learn/validator-trust-score.md

```diff
@@ -0,0 +1,273 @@
+# Nimiq's Validator Trust Score
+
+An algorithm that calculates a score to help users assess how reliable a validator is.{.subline .mb-32}
+
+We will use this score in the Nimiq Wallet to help users decide which validator to trust. The score ranges from 0 to 1, where 0 is not trustworthy at all and 1 is highly trustworthy.
+
+<figure>
+
+<img src="/assets/images/learn/validators-trust-score-wallet-preview.png" alt="The Validator Trust Score in the Wallet" max-h-512 object-contain />
+
+<figcaption>
+
+Preview of the Validator Trust Score in the Nimiq Wallet
+
+</figcaption>
+
+</figure>
+
+## Open source
+
+The algorithm, like the blockchain itself, will be open source. This includes its design as well as its implementation. The implementation is currently under development and will be available as an `npm` package.
+
+An API _may_ also be made available for public use to access the score. More information about this API will be provided in the future.
+
+## The VTS algorithm
+
+The algorithm uses three factors: Size, Reliability, and Liveness. Each factor ranges from 0 to 1.
+
+$$
+T = S \times R \times L
+$$
+
+The Size factor is based on the size of the validator's stake relative to the total stake in the network.
+
+Reliability and Liveness are based on behaviour over the last 9 months. For these parameters we only consider completed epochs, not the currently active one. Therefore, the score is not live and can have a delay of up to 12 hours (an epoch lasts 12 hours).
+
+Before going any further, we define $m$, the number of epochs to consider, knowing that the duration of the window is 9 months.
+
+<details>
+
+<summary children:mx-0>
+
+Calculation of $m$
+
+</summary>
+
+$$
+m = \frac{\text{window\_duration\_ms}}{\text{epoch\_duration}}
+$$
+
+$$
+\text{window\_duration\_ms} = 9 \times 30 \times 24 \times 60 \times 60 \times 1000
+$$
+
+$$
+\text{epoch\_duration} = \text{block\_duration} \times \text{blocks\_per\_epoch}
+$$
+
+  - Block duration and blocks per epoch are constants from the [policy](https://github.com/nimiq/core-rs-albatross/blob/albatross/primitives/src/policy.rs)
+
+</details>
+
+<Callout type="info" no-title>
+
+The curves and constants presented in this document are subject to change at any time in the future. We will keep the community informed of any changes.
+
+</Callout>
+
+Let's look at how each parameter is calculated:
```

Comment:
I think this sentence is unnecessary because the H3 already alludes that we are going to cover the factors